x-pack/plugins/fleet/public/applications/fleet/sections/agents/components/agent_policy_package_badges.tsx

```diff
@@ -71,7 +71,18 @@ export const AgentPolicyPackageBadges: React.FunctionComponent<Props> = ({
           <EuiBadge key={idx} color="hollow">
             <EuiFlexGroup direction="row" gutterSize="xs" alignItems="center">
               <EuiFlexItem grow={false}>
-                <PackageIcon packageName={pkg.name} version={pkg.version} size="s" tryApi={true} />
+                <PackageIcon
+                  packageName={pkg.name}
+                  version={pkg.version}
+                  tryApi={true}
+                  style={
+                    // when a custom SVG is used the logo is rendered with <img class="euiIcon euiIcon--small">
+                    // this collides with some EuiText (+img) CSS from the EuiIcon component
+                    // which  makes the button large, wide, and poorly layed out
+                    // override those styles until the bug is fixed or we find a better approach
+                    { margin: 'unset', width: '16px' }
```

Comment:
do we have to specify  a width in px can we do this by specifying  `width:unset` and using `size="s"` ?

Reply:
I don't believe so but I will double check. I tried that initially, but at a different location, and it didn't work

Reply:
I think there are some larger questions about how to best display the logos. Making everything a 16x16 square makes the landscape orientation logos like netscout, office365 and others very small. However, mixing square and landscape is also not ideal (ragged right edge in drop-down, inconsistent badge dimensions, etc)